### PR TITLE
 Adapt for BR now supporting Negro Leagues

### PR DESCRIPTION
**Pull request by TheCleric**

### CODE REVIEW

File: pybaseball/team_results.py [hunk 1]
```diff
@@ -64,14 +66,14 @@ def get_table(soup,team):
...
-    data = data.reindex(data.index.drop(0))
-    data = data.drop('',1) #not a useful column
-    data['Attendance'].replace(r'^Unknown$', np.nan, regex=True, inplace = True) # make this a NaN so the column can benumeric
-    return data
+    dataframe = pd.DataFrame(data)
+    dataframe = dataframe.rename(columns=dataframe.iloc[0])
+    dataframe = dataframe.reindex(dataframe.index.drop(0))
+    dataframe = dataframe.drop('',1) #not a useful column
+    dataframe['Attendance'].replace(r'^Unknown$', np.nan, regex=True, inplace = True) # make this a NaN so the column can benumeric
+    return dataframe
```
Comment (schorrm) [inline pybaseball/team_results.py:74]: `df`

File: pybaseball/utils.py [hunk 2]
```diff
@@ -13,32 +13,120 @@
...
+# 					'OLY': 1871, 'PBB': 1890, 'PBS': 1914, 'PHA': 1882,
+# 					'PHI': 1873, 'PHK': 1884, 'PHQ': 1890, 'PIT': 1882,
+# 					'PRO': 1878, 'RES': 1873, 'RIC': 1884, 'ROC': 1890,
+# 					'ROK': 1871, 'SDP': 1969, 'SEA': 1977, 'SEP': 1969,
+# 					'SFG': 1958, 'SLB': 1902, 'SLM': 1884, 'SLR': 1875,
+# 					'STL': 1875, 'STP': 1884, 'SYR': 1879, 'TBD': 1998,
+# 					'TBR': 2008, 'TEX': 1972, 'TOL': 1884, 'TOR': 1977,
+# 					'TRO': 1871, 'WAS': 1873, 'WES': 1875, 'WHS': 1884,
+# 					'WIL': 1884, 'WOR': 1880, 'WSA': 1961, 'WSH': 1901,
+# 					'WSN': 2005}
```
Comment (schorrm) [inline pybaseball/utils.py:129]: ????

File: tests/pybaseball/test_teamid_lookup.py [hunk 3]
```diff
@@ -14,7 +14,7 @@ def test_team_id_lookup_all() -> None:
     result = result.query('yearID <= 2019')
 
     assert len(result.columns) == 7
-    assert len(result) == 2925
+    assert len(result) == 2875
```
Comment (schorrm) [inline tests/pybaseball/test_teamid_lookup.py:17]: too specific? or not?

File: pybaseball/utils.py (hunk 2 above)
Comment (TheCleric) [reply pybaseball/utils.py:129]: Whoops!

File: tests/pybaseball/test_teamid_lookup.py (hunk 3 above)
Comment (TheCleric) [reply tests/pybaseball/test_teamid_lookup.py]: It isn't for now. But I'll future proof it.

**Pull request merged**